docs/source/data-publishing/ogcapi-features.rst

```diff
@@ -109,7 +109,17 @@ To publish an Elasticsearch index, the following are required in your index:
          id_field: geonameid
          time_field: datetimefield
 
-This provider has the support for the CQL queries as indicated in the table above.
+.. note::
```

Comment:
@tomkralidis good catch! Should this be added to the records provider, as well? https://github.com/geopython/pygeoapi/blob/master/docs/source/data-publishing/ogcapi-records.rst?plain=1#L50

Reply:
Good idea!  PR updated.  I also noticed that the ES provider in records also does transactions, so this has been updated as well in the docs/this PR.